New Bank/NewBank/newbank/server/NewBank.java

```diff
@@ -53,9 +62,25 @@ public synchronized String processRequest(CustomerID customer, String request) {
 				String to = input[3];
 				return moveFunds(customer,amount,from,to);
 			}
+			if(input[0].equals("PAY")) {
+				if(input.length < 4) { // return fail if not enough information is provided
+					return "FAIL123";
```

Comment:
Looks like a debugging/testing output, should just be "FAIL".

Reply:
done